src/components/DialogTitle.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { PropsWithChildren, ReactElement } from "react"
+import { Close } from "@mui/icons-material"
+import { IconButton } from "@mui/material"
+import MuiDialogTitle from "@mui/material/DialogTitle"
+
+interface DialogTitleProps {
+  onClose?: () => void
+}
+
+export default function DialogTitle({
+  children,
+  onClose,
+}: PropsWithChildren<unknown> & DialogTitleProps): ReactElement {
+  return (
+    <MuiDialogTitle>
+      <IconButton
```

Comment:
Since this is absolute, could it overlap with `children` if they were long enough?

Reply:
Ah, Right.
On another branch I'll give close button on the <Dialog/> component since we don't use title on some component (like reward modal), so let me fix this on next branch, if you like. (we need to give higher zIndex for this button than dialog)